### PR TITLE
Added support for local node

### LINKED ISSUE #4: Add ability to create a Hedera Client for local node
Right now there is a possibility to create client for mainnet | testnet | previewnet

### PR DESCRIPTION
**Pull request by BartoszSolkaArianelabs**
Fixes #4

### CODE REVIEW

File: src/nftSDKFunctions/log-in.ts [hunk 1]
```diff
@@ -18,11 +18,33 @@
...
-  return Client.forName(network).setOperator(myAccountId, myPrivateKey);
+
+  if (network == 'localnode') {
+    return handleLocalNode({ myAccountId, myPrivateKey, network, localNode, localMirrorNode });
+  } else {
+    return Client.forName(network).setOperator(myAccountId, myPrivateKey);
+  }
 };
+
+function handleLocalNode(config: LogInType): Client {
```
Comment (michalrozekariane) [inline src/nftSDKFunctions/log-in.ts:35]: I think it will be better to keep the convention of creating new functions as arrow functions

Comment (StanislawMazowieckiAriane) [reply src/nftSDKFunctions/log-in.ts]: Arrow function +1

File: src/nftSDKFunctions/log-in.ts [hunk 2]
```diff
@@ -18,11 +18,33 @@
...
+  let client: Client;
+
+  if (config.localNode) {
```
Comment (StanislawMazowieckiAriane) [inline src/nftSDKFunctions/log-in.ts:38]: Can be shorten to:

let client: Client = config.localNode ? Client.forNetwork(config.localNode) : Client.forLocalNode();

**Review by michalrozekariane — APPROVED** [review]

File: src/nftSDKFunctions/log-in.ts (hunk 1 above)
Comment (BartoszSolkaArianelabs) [reply src/nftSDKFunctions/log-in.ts]: done

**Review by StanislawMazowieckiAriane — CHANGES_REQUESTED** [review]

File: src/nftSDKFunctions/log-in.ts (hunk 2 above)
Comment (BartoszSolkaArianelabs) [reply src/nftSDKFunctions/log-in.ts]: done

**Review by StanislawMazowieckiAriane — APPROVED** [review]

**Pull request merged**